docs/API/input.md

```diff
@@ -0,0 +1,48 @@
+![Input](https://i.imgur.com/Zm7MTQD.png)
+
+```js
+
+import Icon from 'react-native-vector-icons/FontAwesome';
+import { Input } from 'react-native-elements';
+
+<Input
+  placeholder='BASIC INPUT'
+/>
+
+<Input
+  placeholder='INPUT WITH ICON'
+  icon={
+    <Icon
+      name='user'
+      size={24}
+      color='black'
+    />
+  }
+/>
+
+<Input
+  placeholder='INPUT WITH SHAKING EFFECT'
+  shake={true}
+/>
+
+<Input
+  placeholder='INPUT WITH ERROR MESSAGE'
+  displayError={true}
+  errorStyle={{ color: 'red' }}
+  errorMessage='ENTER A VALID ERROR HERE'
+/>
+
```

Comment:
we should add 1-2 more complex examples here. Look at the login screens on the example app and copy and paste an entire input component with error validation, keyboard type containerStyle etc.

Ping me if you don't know what i am talking about.

Reply:
Understand you perfectly. Going to tackle this in a future update to docs cause I wan't to change the layout without how we have examples and the preview of them